src/Root.tsx

```diff
@@ -0,0 +1,30 @@
+import {
+  HashRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import { App } from './App';
+import { HomePage } from './components/Loader/HomePage';
+import { PeoplePage } from './components/Loader/PeoplePage';
+import React from 'react';
+
+export const Root = () => {
+  return (
+    <Router>
+      <Routes>
+        <Route path="/" element={<App />}>
+          <Route index element={<HomePage />} />
+          <Route path="home" element={<Navigate to="/" replace />} />
+
+          <Route path="people">
+            <Route index element={<PeoplePage />} />
+            <Route path=":slugParam" element={<PeoplePage />} />
+          </Route>
+
+          <Route path="*" element={<h1 className="title">Page not found</h1>} />
```

Comment:
you should create the page

Implement `NotFoundPage` with a title `Page not found` that is shown for all the other URLs;

Reply:
Good point, let's fix it